css/index.css

```diff
@@ -33,22 +43,31 @@ a{
 /* - - - - - H E A D E R - - - - - */
 header{
 	color: black;
-	overflow: hidden;
```

Comment:
I need that, instead of shrinking or enlarging the header, it is its content that adapts to the size, and any element that exceeds it is hidden with this property. Is it possible to leave this property as it was?

Reply:
If the content exceed the size of the box there is lost content, just like the nav bar  in the header in some cases. This is the for the same issue: <https://github.com/lisandroveron/codoacodonews/pull/13/files/6569cd5cb03b8b28ca4a87d6e57e91f53f285b2d#r975674875>